src/query_plan.c

```diff
@@ -297,11 +297,11 @@ QueryPlan *Query_BuildPlan(RedisSearchCtx *ctx, QueryParseCtx *parsedQuery, RSSe
                               Query_OnReopen, plan, NULL, ConcurrentKey_SharedKeyString);
     }
   }
-  if (!parsedQuery || !queryPlan_ValidateQuery(parsedQuery, err)) {
+  if (parsedQuery && !queryPlan_ValidateQuery(parsedQuery, err)) {
```

Comment:
Where do we handle the case where the query is actually NULL?

Reply:
The only case where it is NULL its on coordinator where we are not parsing the query and this is ok to just ignore it.
If you check I changed it because I thought it was wrong and it cause a bug in the coordinator so I changed it back ...